### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by robvanvolt** [lucidrains/DALLE-pytorch#50]
I myself have trained a solid vae-model and am currently working on the dalle-model. How come noone has posted a pretrained model yet? It seems like a waste of ressources if everyone has to start vom scratch?

I'm on day 3 of training and still at a loss of 2,5 - i will have to increase the total number of images and include sparse attention next, as the 2,5 looks like a plateu with the current settings.

**Comment by robvanvolt** [comment]
A pretrained model got released yesterday by the official DALL-E pytorch github repository, lucidrains already included it in the latest update!
